src/SWP/Bundle/ContentBundle/Doctrine/ODM/PHPCR/ArticleRepository.php

```diff
@@ -34,4 +38,26 @@ public function findAllArticles()
     {
         return $this->createQueryBuilder('o')->getQuery();
     }
+
+    public function getQueryForRouteArticles(string $identifier, array $order) : SqlQuery
```

Comment:
what about also changing other functions signatures to PHP 7 syntax? because one method is php 7 syntax and other are not, I would make it the same